Tests/Unit/EmogrifierTest.php

```diff
@@ -1848,7 +1891,8 @@ public function dataUrisAreConserved()
     {
         $html = $this->html5DocumentType . '<html></html>';
         $this->subject->setHtml($html);
-        $styleRule = 'background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAIAAAAC64paAAABUk' .
+        $styleRule = 'background-image: url(data:image/png;charset=utf-8;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUC' .
```

Comment:
This looks to be focusing on data uris, whereas the code is just providing some exceptions when splitting on ';'. I think that the current test is sufficient for now. If in the future other special cases arise then they would need to be added.